oap-server/server-bootstrap/src/main/resources/metadata-service-mapping.yaml

```diff
@@ -13,5 +13,5 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-serviceName: ${LABELS."service.istio.io/canonical-revision"}.${LABELS."service.istio.io/canonical-name",LABELS."app.kubernetes.io/name",LABELS.app}.${NAMESPACE}
+serviceName: ${LABELS."service.istio.io/canonical-name",LABELS."app.kubernetes.io/name",LABELS.app}
```

Comment:
What's the reason to change this back and forth? I remember you change this in the previous PR

Reply:
to keep this consistent with the default ALS `k8sServiceNameRule` by default, and can make the topology consistently if ALS(k8s-mesh) and envoy metrics work at the same time.